scripts/fix_files.sh

```diff
@@ -81,8 +81,9 @@ if [[ $do_clang_format -eq 1 && -f ./.clang-format ]]; then
 	if [ $? -eq 0 ]; then
 		cf_version="$(clang-format --version)"
 		regex="clang-format version 7\.0\.0 \(tags\/RELEASE_700\/final[ 0-9]*\/WithWrappingBeforeLambdaBodyPatch\)"
-		if [[ ! "$cf_version" =~ $regex ]]; then
-			echo "Incorrect clang-format: Version 7.0.0 with WrappingBeforeLambdaBody patch required (found: $cf_version)"
+		regex2="clang-format version 1[1-9]\.[ 0-9]*"
```

Comment:
Until I'm satisfied with a recent version of clang-format, I won't be able to allow a version change.
Also, I'd rather have a strict version check for clang-formatting as any new version might introduce slight changes in the formatting (like possible bug fix), and if not all developers use the exact same version, each commit from each developer will possibly introduce formatting diff.

Reply:
to my mind, forcing developers to use patched legacy version is much more bad idea, than sticking with freshly available mainline one. What do you think?

Reply:
Consistency is far better than chaos :)
Especially when it comes to formatting code (visual change) being mixed with actual code change.
I provide pre-built versions and the patch in case someone don't trust.

I'd fully support a more recent version of clang-format as soon as the downsides compensate the upsides.
In any case, forcing the version is (for me) the best way to go, as it imposes consistency.
If we just say "use any version greater than clang-format vX.Y" then we cannot be sure that at some point in the future the default behaviour of an external tool won't change (this includes any bug added or fixed).
And it already is the case between the version you used and mine (a fix was made to .mm files, and to #pragma directives), so we already have examples.